### PR TITLE
RST-3466 rails update

### PR DESCRIPTION
**Pull request by zaparka**
Rails 7 update branch

### CODE REVIEW

File: config/initializers/new_framework_defaults_6_1.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+#
+# This change is not backwards compatible with earlier Rails versions.
+# It's best enabled when your entire app is migrated and stable on 6.1.
+# Rails.application.config.action_dispatch.cookies_same_site_protection = :lax
+
+# Generate CSRF tokens that are encoded in URL-safe Base64.
+#
+# This change is not backwards compatible with earlier Rails versions.
+# It's best enabled when your entire app is migrated and stable on 6.1.
+Rails.application.config.action_controller.urlsafe_csrf_tokens = true
```
Comment (WillTaylor22) [inline config/initializers/new_framework_defaults_6_1.rb:32]: Is the rest of this file needed? And should it not be merged into the new_framework_defaults_7_*?

File: config/initializers/new_framework_defaults_7_0.rb [hunk 2]
```diff
@@ -0,0 +1,117 @@
+# Be sure to restart your server when you modify this file.
```
Comment (WillTaylor22) [inline config/initializers/new_framework_defaults_7_0.rb:1]: Is this file needed?

File: config/initializers/permissions_policy.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Define an application-wide HTTP permissions policy. For further
```
Comment (WillTaylor22) [inline config/initializers/permissions_policy.rb:1]: Is this file needed?

**Review by WillTaylor22 — APPROVED** [review]

File: config/initializers/new_framework_defaults_7_0.rb (hunk 2 above)
Comment (zaparka) [reply config/initializers/new_framework_defaults_7_0.rb:1]: Yeah I would like to keep it there until next rails update.
